Code/GraphMol/FileParsers/file_parsers_catch.cpp

```diff
@@ -3019,6 +3020,47 @@ TEST_CASE("test bond flavors when writing PDBs", "[bug]") {
   }
 }
 
+TEST_CASE("test output with incomplete monomer info", "[bug][writer]") {
+  SECTION("basics") {
+    {
+      std::unique_ptr<RWMol> m{SmilesToMol("Cl")};
```

Comment:
constructing a molecule safely from an explicit SMILES is much simpler as:
```suggestion
      auto m = "Cl"_smiles;
```

This suggestion applies to all of the molecule construction below too.

Reply:
Ok, I changed that